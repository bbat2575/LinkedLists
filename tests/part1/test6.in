NEW 5
2e0
2E1
2e2
2E5
2e10
TYPE 0
NEW 5
2E+0
2e+1
2E+2
2e+5
2E+10
TYPE 1
NEW 5
2E-0
2e-1
2E-2
2e-5
2E-10
TYPE 2
NEW 6
+2e0
+2e2
-2e0
-2e2
+2e-1
-2e+1
TYPE 3
NEW 8
2e0000
0002e2
2e+0000
+0002e2
2e-0000
-0002e2
+0002e-2
-2e+0000
TYPE 4
NEW 15 
e
E
eE
e0
0e
e2
2e
2e+
2e-
+2e
-2e
2ee
ee2
2ee2
e2e
TYPE 5