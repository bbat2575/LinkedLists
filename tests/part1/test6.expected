
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 2.00 -> 20.00 -> 200.00 -> 200000.00 -> 20000000000.00

> float -> float -> float -> float -> float

> List 1: 2.00 -> 20.00 -> 200.00 -> 200000.00 -> 20000000000.00

> float -> float -> float -> float -> float

> List 2: 2.00 -> 0.20 -> 0.02 -> 0.00 -> 0.00

> float -> float -> float -> float -> float

> List 3: 2.00 -> 200.00 -> -2.00 -> -200.00 -> 0.20 -> -20.00

> float -> float -> float -> float -> float -> float

> List 4: 2.00 -> 200.00 -> 2.00 -> 200.00 -> 2.00 -> -200.00 -> 0.02 -> -2.00

> float -> float -> float -> float -> float -> float -> float -> float

> List 5: e -> E -> eE -> 0.00 -> 0.00 -> 0.00 -> 2.00 -> 2.00 -> 2.00 -> 2.00 -> -2.00 -> 2ee -> ee2 -> 2ee2 -> e2e

> char -> char -> string -> float -> float -> float -> float -> float -> float -> float -> float -> string -> string -> string -> string

> 
